cmd/dlt/service/cmd.go

```diff
@@ -0,0 +1,124 @@
+/*
+Copyright (c) 2020 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package service
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	msv1 "github.com/openshift-online/ocm-sdk-go/servicemgmt/v1"
+	"github.com/openshift/rosa/pkg/interactive/confirm"
+	"github.com/openshift/rosa/pkg/logging"
+	"github.com/openshift/rosa/pkg/ocm"
+	rprtr "github.com/openshift/rosa/pkg/reporter"
+)
+
+var args struct {
+	// ID of service
+	ID string
+}
+
+var Cmd = &cobra.Command{
+	Use:   "service",
+	Short: "Deletes a service",
+	Long:  "Deletes a service.",
+	Example: `  # Delete a service with ID "aabbcc"
+  rosa delete service --id=aabbcc`,
+	Run:    run,
+	Hidden: true,
+}
+
+func init() {
+	flags := Cmd.Flags()
+
+	flags.StringVar(&args.ID,
+		"id",
+		"",
+		"The ID of the service to be deleted.")
+}
+
+func run(cmd *cobra.Command, _ []string) {
+	reporter := rprtr.CreateReporterOrExit()
+	logger := logging.CreateLoggerOrExit(reporter)
+
+	if !confirm.Confirm("delete service with id '%s'", args.ID) {
```

Comment:
I think we should make arge.ID required or add a nil check, to save the following error when no id is passed 
```
rosa delete service 
? Are you sure you want to delete service with id ''? Yes
E: expected response content type 'application/json' but received '' and content ''

```

Reply:
https://github.com/openshift/rosa/pull/654/commits/93d12bd3a5c5a7baba448edd030c74e32be47baa#diff-8c41d30fb00687e5de664bfb4cb8fab7a8bb0fac106c9b2c677049d600ce0d1fR61